build/debs/BUILD

```diff
@@ -110,6 +110,24 @@ genrule(
     tags = ["manual"],
 ) for arch in NODE_PLATFORMS["linux"]]
 
+[deb_data(
+    name = "cri-dockerd",
+    data = [
+        {
+            "files": ["//cmd/cri-dockerd"],
+            "mode": "0755",
+            "dir": "/usr/bin",
+        },
+        {
+            "files": ["cri-dockerd.service"],
```

Comment:
I don't see this file in the PR. Perhaps it's missing or uncommitted?
Anyway, I presume it's going to be roughly equivalent to kubelet's systemd service file.

Reply:
Hmm, i missed this, will add one